src/js/utils/accountUtils.js

```diff
@@ -27,6 +27,9 @@ export const getAccountAddressN = (
     if (coinInfo.type === 'bitcoin') {
         return [toHardened(options.purpose), toHardened(options.coinType), toHardened(index)];
     }
+    if (coinInfo.shortcut === 'ADA') {
+        return [toHardened(1852), toHardened(coinInfo.slip44), toHardened(index)];
```

Comment:
```suggestion
        return [toHardened(options.purpose), toHardened(coinInfo.slip44), toHardened(index)];
```
purpose should be: 44 or 48 or 84 [read more about bip44](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki#purpose) and get inspiration from cardanoGetPublicKey [tests](https://github.com/trezor/connect/blob/develop/tests/__fixtures__/cardanoGetPublicKey.js) 
In your case, since you are not passing any custom `bip44` in params - default value will be `44` (line 22)
`1852` is cardano `coinInfo.slip44` value (bip44 coin_type)

Reply:
in fact you dont have to do any changes here, all `misc` coins should be covered in line 41 below

Reply:
This is actually correct, 44 stands for Byron (legacy) addresses. see: https://github.com/cardano-foundation/CIPs/blob/master/CIP-1852/CIP-1852.md

Reply:
oh, i see. so could you please add a comment there with cip1852 link to make it obvious why it's hardcoded?